src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1742,7 +1746,11 @@ public String getQuestionText(String feedbackQuestionId) {
         return sortedMap;
     }
 
-    public LinkedHashMap<String, Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>>
+    /**
+     * Returns an ordered Map with {@code giverTeam} name as key
+     * sorted by {@code compareByTeamQuestionGiverTeamRecipient} comparator.
```

Comment:
This is also a private comparator. Change to `... sorted by giverTeam > question > giverName > recipientTeam > recipientName` @wkk91193